### PR TITLE
reset frozen mathversion for #1028

### PR DESCRIPTION
**Pull request by davidcarlisle**
Possible fix for #1028

# Internal housekeeping

## Status of pull request

- Feedback wanted 
I didn't add rollback, if it is needed I guess it should have the original from 0000/00/00 and this from 2022/11/01 (when freeze mechanism added)  but this version should work with old formats anyway so no rollback seems ok?

## Checklist of required changes before merge will be approved
- [X] Test file(s) added
- [X] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/ltfssbas.dtx [hunk 1]
```diff
@@ -1050,11 +1050,22 @@
 %      {\cs{def} $\to$ \cs{edef} for \cs{math@version}.}
 % \changes{v1.2g}{1990/02/16}{\cs{@nomath} added.}
 % \changes{v2.1a}{1994/01/17}{New math font setup}
+% \changes{v3.2l}{2023/04/01}{Reset frozen mathversion gh/1028}
 %    \begin{macrocode}
 \DeclareRobustCommand\mathversion[1]
          {\@nomath\mathversion
           \expandafter\ifx\csname mv@#1\endcsname\relax
           \@latex@error{Math version `#1' is not defined}\@eha\else
+%    \end{macrocodpe}
```
Comment (FrankMittelbach) [inline base/ltfssbas.dtx:1059]: why didn't that fail?

**Review by FrankMittelbach — APPROVED** [review]
probably doesn't need  an entry in ltnews but it wouldn't hurt either I guess

and the \end... should be fixed, otherwise fine I think

File: base/ltfssbas.dtx (hunk 1 above)
Comment (davidcarlisle) [reply base/ltfssbas.dtx:1059]: my typing doesn't improve... It doesn't generate an error, just matches the next one.

**Review by FrankMittelbach — APPROVED** [review]
looks good now

**Pull request merged**